### ISSUE TITLE
source map not building in --batch

### ISSUE THREAD

**Issue by danny007in** [jakubpawlowicz/clean-css-cli#61]
Actually without --output

**Comment by danny007in** [comment]
```console
> cleancss -O1 --format breakWith=lf --source-map --source-map-inline-sources --with-rebase --batch --batch-suffix .min  "dist/css/*.css" "!dist/css/*.min.css"

ERROR: Source maps will not be built because you have not specified an output file.
```

**Comment by danny007in** [comment]
but with `--output --batch` 
`--with-rebase` not working
💯 that's the conflict

**Comment by jakubpawlowicz** [comment]
Thanks for reporting @danny007in - I'll have a closer look tomorrow.

**Comment by jakubpawlowicz** [comment]
Sorry it's not been addressed yet - I want to include the fix in 5.3 later this week so stay tuned.

**Comment by jakubpawlowicz** [comment]
The version on main branch:

1) doesn't show "ERROR: Source maps will not be built because you have not specified an output file." error and correctly builds source maps in batch mode
2) fixes rebasing when `--output` is a directory

Hope this helps. I'm going to release it in 5.3 tomorrow.
